WTWSMS/events/ACR_celtic_flavor_events.txt

```diff
@@ -409,6 +436,7 @@ character_event = {
 		}
 		hidden_tooltip = {
 			character_event = { id = ancrel.0210 days = 10 }
+			remove_trait = traveling
```

Comment:
It shouldn't be. He TRAVELS to the island and goes on a great adventure to be cured. That is the definition of traveling.